docs/guides/runtime-options-overview.mdx

```diff
@@ -177,9 +189,20 @@ Due to differences in the device compilation process, certain runtime features c
   Can be used with pulse gates, but it might not work when using custom gates.
   </TabItem>
 
+  <TabItem value="Gen3" label="gen3-turbo">
```

Comment:
There is actually another `gen3-turbo` tab already all the way in the end

Reply:
yep, I just noticed that and deleted it so they're alphabetical.